provision/acc_provision/acc_provision.py

```diff
@@ -743,6 +772,20 @@ def config_validate_preexisting(config, prov_apic):
                 warn("L3out not defined in the APIC: %s/%s" %
                      (vrf_tenant, l3out_name))
 
+            # Following code is to detect a lagacy cluster
+            # kube_ap = apic.get_ap(config["aci_config"]["system_id"])
+            # if an app profile with the name "kubernetes" exists under system
+            # tenant, this means the cluster was provisioned with older
+            # naming convention. This is a fallback in case the user
+            # forgets to add the field to indicate an existing legacy
+            # cluster.
+            # if kube_ap:
+            #     config["aci_config"]["use_legacy_kube_naming_convention"] = True
+            #     if config["aci_config"]["vmm_domain"]["type"] == "OpenShift":
+            #         config["kube_config"]["system_namespace"] = "aci-containers-system"
+            #     else:
+            #         config["kube_config"]["system_namespace"] = "kube-system"
+
```

Comment:
do we need this? (btw, minor typo: "lagacy")

Reply:
Commented out for now so that it can be added to the code later.